apps/newsletters-ui/src/app/components/views/DraftListView.tsx

```diff
@@ -1,31 +1,35 @@
 import AddIcon from '@mui/icons-material/Add';
-import { Box, Container } from '@mui/material';
-import { useLoaderData } from 'react-router-dom';
+import { Box, Button, Container } from '@mui/material';
+import { useLoaderData, useNavigate } from 'react-router-dom';
 import { isDraft } from '@newsletters-nx/newsletters-data-client';
 import { DraftsTable } from '../DraftsTable';
-import { NavigateFab } from '../NavigateFab';
```

Comment:
May as well delete `NavigateFab` if we aren't using it here - this is the only page it was on. It was basically a copy of the NavigateButton, but use `Fab` instead of `Button` as the base.